src-v2/Judge/NTS.Judge.Blazor/Pages/Dashboard/Components/Actions/EliminationForms/FailedToQualifyForm.razor

```diff
@@ -34,69 +33,57 @@
     [Parameter]
     public FailedToQualify? FailedToQualify { get; set; }
     [Parameter]
-    public EventCallback OnSubmission { get; set; }
+    public EventCallback OnRestore { get; set; }
+    [Parameter]
+    public EventCallback OnEliminate { get; set; }
 
-    private string? FTQReason
+    private IEnumerable<string>? FTQReason { get; set; }
+    public string? Reason { get; set; }
+    private bool _reasonInputState;
```

Comment:
Is this necessary? Why not simple render reason input if there is `Complement` value?